packages/bootstrap-vue-next/src/components/BForm/BFormDatalist.vue

```diff
@@ -3,11 +3,7 @@
     <slot name="first" />
     <template v-for="(option, index) in normalizedOptsWrapper" :key="index">
       <BFormSelectOption :value="option.value" :disabled="option.disabled">
-        <!-- eslint-disable-next-line vue/no-v-html -->
-        <span v-if="!!option.html" v-html="option.html" />
-        <template v-else>
-          {{ option.text }}
-        </template>
+        {{ option.text }}
```

Comment:
```suggestion
        <slot v-bind="option">
          {{ option.text }}
        </slot>
```

and then change the slot below to name="last" Or this part to name="option"